### ISSUE TITLE
The counter label in Traditional Chinese is overlapped with the counter

### ISSUE THREAD

**Issue by aethanyc** [mozilla/fx-private-relay#2575]
There are three counters, and their text are displayed vertically due to insufficient width. The third one (number of tracker removed?) are especially longer and overlapped with the counter number.

![broken](https://user-images.githubusercontent.com/102858/192393743-0eca6051-e23f-4d62-8845-25135b988653.png)

**Comment by aethanyc** [comment]
The text looks good to me by either setting `word-break:keep-all` on `.Alias_label__JAlvU`, or removing `flex-basis: 0` on `.Alias_alias-stats__ms2tv`.

![good](https://user-images.githubusercontent.com/102858/192394606-dbc6e62d-799a-45fa-9379-86f0f33c739e.png)

**Comment by Vinnl** [comment]
Thanks @aethanyc, both for reporting and suggesting a fix! Just letting you know that we're a bit busy this week, but we'll look at this as soon as possible. Finding issues like this in all languages we support can be challenging, so reports like this are much appreciated!

**Comment by Vinnl** [comment]
Hi @aethanyc, I've got a fix lined up in https://github.com/mozilla/fx-private-relay/pull/2666. Would you mind checking out [the preview](https://deploy-preview-2666--fx-relay-demo.netlify.app/accounts/profile/?mockId=some), to verify that it does indeed fix the issue? Thanks!

**Comment by aethanyc** [comment]
Hi @Vinnl, thanks for the fix! The preview looks good on my macOS, but on my linux desktop, the text label can still display vertically or being wrapped to second line due to small width. That might related to the font I used. I check via Firefox devtools font panel, it is "Noto Sans CJK TC". Here is what I see.
![Screenshot 2022-10-17 at 14-33-19 Firefox Relay](https://user-images.githubusercontent.com/102858/196288816-67b980c8-c52a-4c0b-a308-8eeb18e751a1.png)

To prevent CJK text from breaking while preserving non-CJK text's behavior, [`word-break: keep-all`](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break) might be helpful.

**Comment by Vinnl** [comment]
@aethanyc Hmm... OK, so the problem here is that in other languages, we _do_ want a word break, e.g. in English, we want "Trackers Removed" to span two lines:

![image](https://user-images.githubusercontent.com/4251/196151386-fbb32790-a821-4532-a6aa-92ff3c6351e7.png)

Is there any way that that would make sense in the Chinese version? i.e. what would happen if that label was printed on a very thin strip of paper?

**Comment by aethanyc** [comment]
> @aethanyc Hmm... OK, so the problem here is that in other languages, we do want a word break, e.g. in English, we want "Trackers Removed" to span two lines:

You're right, and this is why I feel `word-break: keep-all` [1] might be useful here, because it only affects CJK, but not other western languages that use space and punctuation as the line break opportunities. So "Trackers Removed" can still span two lines with `word-break: keep-all` applied.

[1] Quote the [CSS text spec](https://drafts.csswg.org/css-text-3/#word-break-property) text: "This property (word-break) specifies [soft wrap opportunities](https://w3c.github.io/csswg-drafts/css-text/#soft-wrap-opportunity) between letters, i.e. where it is “normal” and permissible to break lines of text. ... It does not affect rules governing the soft wrap opportunities created by [white space](https://w3c.github.io/csswg-drafts/css-text/#white-space) (as well as by [other space separators](https://w3c.github.io/csswg-drafts/css-text/#other-space-separators)) and around punctuation." 

> Is there any way that a line break would make sense in the Chinese version? i.e. what would happen if that label was printed on a very thin strip of paper?

A line break can occur anywhere in Chinese letters, so by default layout just allows it, and CSS invented `word-break` to control alternate line-breaking behavior between letters. For the text "封鎖" ("Blocked"), it makes sense that the line breaks in between "封" and "鎖" in a very small width div or on a thin strip of paper. It's just that on this particular UI, breaking the line in between "封鎖" doesn't look natural since there is a lot of blank areas around the text on the screen.

**Comment by Vinnl** [comment]
Thanks @aethanyc! I had actually tried it, but without also changing the `flex-basis` of `.alias-stats` to `auto`, and that's why `keep-all` didn't initially add a line break in between "Trackers Removed". After applying both changes, it does indeed now display as I'd expect it (a single line in Chinese, two lines in English). That means [the preview](https://deploy-preview-2666--fx-relay-demo.netlify.app/accounts/profile/?mockId=some) should now look better for you on both MacOS and Linux. Thank you for all the input, that was very helpful, and I learned something about dealing with CJK text :)

**Comment by aethanyc** [comment]
Thank you for fixing this issue @Vinnl! [The preview](https://deploy-preview-2666--fx-relay-demo.netlify.app/accounts/profile/?mockId=some) looks good to me on both macOS and Linux 🎉.
